README.md

```diff
@@ -68,6 +68,7 @@ Examples of Slate in the Wild
 * [Dwolla API docs](https://docs.dwolla.com/)
 * [RozpisyZapasu API docs](http://www.rozpisyzapasu.cz/dev/api/)
 * [Codestar Framework Docs](http://codestarframework.com/documentation/)
+* [Buddycloud API](http://buddycloud.com/api/)
```

Comment:
Heh, one more. The trailing slash actually breaks this links. For some reason I only get a GitHub error page.